hummingbot/connector/derivative/orderly_perpetual/orderly_perpetual_derivative.py

```
@@ -0,0 +1,1864 @@
+- Order placement and management
+- Position tracking
+- Balance management
+- Funding rate information
+- Real-time market data via WebSocket
+
+Reference: Orderly Network EVM API
+https://orderly.network/docs/build-on-omnichain/evm-api/introduction
+"""
+
+import asyncio
+import json
```

Flake8 (7.3.0)

[error] 'json' imported but unused

(F401)